dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230707CreateSystemTable.java

```diff
@@ -0,0 +1,33 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import io.vavr.control.Try;
+
+/**
+ * Creates the system table
+ * @author jsanca
+ */
+public class Task230707CreateSystemTable implements StartupTask {
+
+    @Override
+    public boolean forceRun() {
+        //
+        return Try.of(()->!new DotDatabaseMetaData().tableExists(
+                DbConnectionFactory.getConnection(), "system_table")).getOrElse(true);
+    }
+
+    @Override
+    public void executeUpgrade() throws DotDataException, DotRuntimeException {
+
+        Try.of(()->new DotConnect().executeStatement("CREATE TABLE system_table ("
```

Comment:
We should always add `IF NOT EXISTS ` when adding tables, e.g. `CREATE TABLE IF NOT EXISTS  system_table`

Reply:
thinking about 2 nodes coming up up a cluster simultaneously.